patroni/ctl.py

```diff
@@ -21,6 +20,10 @@
 import tempfile
 import time
 import yaml
+try:
+    import ydiff
+except ImportError:
+    import cdiff as ydiff
```

Comment:
```suggestion
try:
    from ydiff import markup_to_pager, PatchStream
except ImportError:  # pragma: no cover
    from cdiff import markup_to_pager, PatchStream
```

And please move it after the `from six.moves.urllib_parse import urlparse`